app/src/renderer/connectors/lcdClientMock.js

```diff
@@ -1184,8 +1184,11 @@ module.exports = {
     results.push(txResult(0))
     return results
   },
-  async getProposalVotes(proposalId) {
-    return state.votes[proposalId] || []
+  async queryProposalVotes(proposalId) {
+    return (
+      state.votes[proposalId] ||
+      Promise.reject(`Invalid proposalId ${proposalId}`)
```

Comment:
we should use the error structure format from the SDK. I'd propose to leave it now as it is and modify this everywhere on another PR

Reply:
Agree! I just tried locally and currently:

https://localhost:9070/gov/proposals/3/votes => null
https://localhost:9070/gov/proposals/1/votes => [{proper array of votes}]

So instead of that message I should place the same value I've got from the sdk ok?

Reply:
Ok we are currently missing a proper error message there, I will leave this message but wrapped in the proper object:

https://github.com/cosmos/cosmos-sdk/issues/3078